functions/tokenservice/functions/package.json

```diff
@@ -5,19 +5,19 @@
     "lint": "eslint .",
     "serve": "firebase serve --only functions",
     "shell": "firebase functions:shell",
-    "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
     "local-test": "mocha test/index.test.js",
     "pretest": "sh test/updateFunctions.sh",
```

Comment:
Delete this line if you delete `updateFunctions.sh`.

Reply:
Done.